cmd/cni-metrics-helper/metrics/metrics.go

```diff
@@ -397,13 +401,13 @@ func metricsListGrabAggregateConvert(t metricsTarget) (map[string]*dto.MetricFam
 	resetMetrics(interestingMetrics)
 
 	targetList := t.getTargetList()
-	glog.Info("targetList: ", targetList)
-	glog.Info("len(targetList)", len(targetList))
+	log.Infof("targetList: %v ", targetList)
+	log.Infof("len(targetList) %d", len(targetList))
```

Comment:
The above log messages are not needed. They are really just temporary debugging prints and don't belong as log messages.